### DISCUSSION TITLE
Implement dunder `repr` for classes

### DISCUSSION THREAD

**Discussion by fjuniorr** [manusimidt/py-xbrl#100] [Ideas]
Thanks for the great package @manusimidt!

How do you feel about adding [`__repr__` methods](https://docs.python.org/3/library/functions.html#repr) to all classes so that's easier to explore a new instance? I would be happy to submit a PR with a shot at it if you feel the idea is worthwhile.

**Comment by manusimidt** [comment]
Hey @fjuniorr 

yes, sounds like a good idea! I have only implemented `__str__` methods in most of the classes. 
if you want you are welcome to do a PR :) 
Otherwise, I will add it to my list of future changes and features.
